daemon/algod/api/algod.oas2.json

```diff
@@ -3648,6 +3648,10 @@
       "description": "The set of parameters and limits override during simulation. If this set of parameters is present, then evaluation parameters may differ from standard evaluation in certain ways.",
       "type": "object",
       "properties": {
+        "signatures-optional": {
+          "description": "If true, transactions without signatures are allowed and treated as if they were properly signed.",
```

Comment:
```suggestion
          "description": "If true, transactions without signatures are allowed and simulated as if they were properly signed.",
```

Reply:
Fixed in a56938b5568a78e69926ae151083bb8ba4c07468